manual/create-repo.md

```diff
@@ -15,6 +15,16 @@
     * **Require pull request reviews before merging** *(default 1 review is OK)*
     * **Dismiss stale pull request approvals when new commits are pushed**
     * **Include administrators**
+    
+    For any repositories that use a multi-branch Jenkins pipeline the following additional branch protection rules should be set for the *master* rule:
+    * **Require status checks to pass before merging**      
+      * **Require branches to be up to date before merging** *(sub option)*
+      
+      Choose the following status checks:
+      * **TDR Jenkins build status** *(make this a required check)*
+      
+      **Note**: the Jenkins job will need to be configured first before this status check will appear.
```

Comment:
A link to an example of this would be good, e.g. to the specific line in the frontend Jenkinsfile.

Reply:
Added an example, but didn't add line numbers as these may change in the future causing confusion. Just referenced the relevant functions in the jenkins file

Reply:
That makes it clear, thanks. 👍